gen/testdata/test.thrift.go

```diff
@@ -365,6 +396,17 @@ func _Event_Read(w wire.Value) (*Event, error) {
 	err := v.FromWire(w)
 	return &v, err
 }
+func (v *Event) String() string {
+	var fs [2]string
+	i := 0
+	if v.Time != nil {
+		fs[i] = fmt.Sprintf("Time: %v", *(v.Time))
```

Comment:
nit: do you need to dereference here? since the `String` method is already on the pointer, it should work without right?